main/main.ino

```diff
@@ -970,6 +970,27 @@ bool wifi_reconnect_bypass() {
   while (WiFi.waitForConnectResult() != WL_CONNECTED && wifi_autoreconnect_cnt < maxConnectionRetryWifi) {
 #  endif
     Log.notice(F("Attempting Wifi connection with saved AP: %d" CR), wifi_autoreconnect_cnt);
+
+    //Reduce WiFi interference when using ESP32 using custom WiFi mode and tx power
+#  ifdef ESP32
+    // https://github.com/jnogues/ESP32-Long-Range-WiFi/issues/4
+    // Reset wifi protocol
+    if (esp_wifi_set_protocol(WIFI_IF_STA, WIFI_PROTOCOL_11B | WIFI_PROTOCOL_11G | WIFI_PROTOCOL_11N | WIFI_PROTOCOL_LR) != ESP_OK) {
+      Log.error(F("Failed to reset WifiMode." CR));
+    }
+//https://github.com/espressif/arduino-esp32/search?q=WIFI_PROTOCOL_11G
+//https://www.letscontrolit.com/forum/viewtopic.php?t=671&start=20
+#    ifdef WifiGMode
+    if (esp_wifi_set_protocol(WIFI_IF_STA, WIFI_PROTOCOL_11B | WIFI_PROTOCOL_11G) != ESP_OK) {
+      Log.error(F("Failed to change WifiMode." CR));
+    }
+#    endif
+
+#    ifdef WifiPower
```

Comment:
Could you add these macros into User_config.h (commented) with an explanation

Reply:
Done.